README.md

```diff
@@ -28,34 +28,105 @@ The ```winetricks``` package should be used if it is available and up to date. T
 
 Note: packaged Debian / Ubuntu winetricks versions are typically outdated, so a manual installation is recommended.
 
-If the package is unavailable, outdated, or the latest version is desired, a manual installation of winetricks can be done. E.g.:
+If the package is unavailable, outdated, or the latest version is desired, a manual installation of winetricks can be done.
+It is recommended to uninstall any previously installed version of winericks first.
 
+**_If you don't uninstall a previously installed, packaged version of winetricks... Well then you get to pick up the pieces!_**
+
+E.g. for Debian / Ubuntu:
+```
+sudo apt-get purge winetricks
 ```
-sudo -- sh -c '
-cat > /usr/local/bin/update_winetricks.sh <<_EOF_SCRIPT
+
+Then, for Ubuntu, use a simple shell script to download the current winetricks scripts.
+This script uses a single parameter (to set a working directory).
+E.g.:
+
+```
+# Change to a directory writeable by current user (their HOME directory). See:
+#   https://www.tldp.org/LDP/abs/html/internalvariables.html
+cd "${HOME}"
+
+# Use a BASH "here document" to create an updater shell script file.
+# See:
+#   https://www.tldp.org/LDP/abs/html/here-docs.html
+# >  outputs stdout to a file, overwriting any pre-existing file
+# << takes input, directly from the script itself, till the second '_EOF_SCRIPT' marker, as stdin
+# the cat command hooks these 2 streams up (stdin and stdout)
+###### create update_winetricks.sh START ########
+cat > update_winetricks.sh <<_EOF_SCRIPT
 #!/bin/sh
 
-cd "${HOME}" # change to a path writeable by current user
+# Change to a directory writeable by current user. See:
+#   https://www.tldp.org/LDP/abs/html/internalvariables.html
+# Use a script parameter $1 or fallback to the user's HOME directory.
+# This works around the lack of env variable support in systemd units (see below).
+cd "\${1:-\${HOME}}"
```

Comment:
Why not just use a tempdir? E.g., `cd $(mktemp -d)`. The user won't need to interact with anything in there anyway, since it's an automated script.